build.sbt

```diff
@@ -112,6 +112,20 @@ lazy val blazeClient = libraryProject("blaze-client")
   )
   .dependsOn(blazeCore % "compile;test->test", client % "compile;test->test")
 
+lazy val mikuServer = libraryProject("miku-server")
```

Comment:
Alright, I'll be the grump here.  Discoverability is important.  Can a user intuit what exists from naming conventions, scanning the directory structure, etc.?

- Does http4s support circe? http4s-circe, `org.http4s.circe`
- Does it support servlets? http4s-servlet, `org.http4s.servlet`  
- Does it support twirl? http4s-twirl, `org.http4s.twirl`
- Does it support netty servers? http4s-miku, `org.http4s.miku` :confused: 

Now, it would be tragic if your ASCII shrine to miku were lost.  Could we leave it as a whimsical banner in the server package so that the glory of Miku could be shared across all backends?  Use it in a tut for how to configure a server?  I'm all for having a laugh, but let's make sure it's not detracting from a high quality and oft-requested module.
 
:notes: I don't think Hank done it this way :notes:

Reply:
Fair. i think I can rename to http4s netty and leave the glory of miku in my PR